Source/Core/VideoCommon/VertexLoaderManager.cpp

```diff
@@ -152,6 +155,19 @@ int RunVertices(int vtx_attr_group, int primitive, int count, DataReader src, bo
 	if (skip_drawing || is_preprocess)
 		return size;
 
+	// Hide Objects Code code
+	if (m_LocalCoreStartupParameter.num_object_removal_codes)
```

Comment:
Well, it saves two "mov" commands, so it's slightly faster, but probably not worth it at this point.  It used to make a lot of sense, because the loop was not as optimized...